frontend/integration-tests/tests/crd-extensions.scenario.ts

```diff
@@ -199,4 +199,91 @@ describe('CRD extensions', () => {
     });
   });
 
+  describe('ConsoleExternalLogLink CRD', () => {
+    const crd = 'ConsoleExternalLogLink';
+    const name = `${testName}-cell`;
+    const podName = `${testName}-pod`;
+    const cell = $(`[data-test-id=${name}]`);
+    const text = `${name} Logs`;
+    const namespaceFilter = '^openshift-';
+
+    it(`displays YAML editor for creating a new ${crd} instance`, async() => {
+      await browser.get(`${appHost}/k8s/cluster/customresourcedefinitions?name=${crd}`);
+      await crudView.isLoaded();
+      await crudView.clickKebabAction(crd, 'View Instances');
+      await crudView.isLoaded();
+      await crudView.createYAMLButton.click();
+      await yamlView.isLoaded();
+      const content = await yamlView.getEditorContent();
+      const newContent = _.defaultsDeep({}, {metadata: {name}, spec: {text}}, safeLoad(content));
+      await yamlView.setEditorContent(safeDump(newContent));
+      expect(yamlView.getEditorContent()).toContain(`kind: ${crd}`);
+    });
+
+    it(`creates a new ${crd} instance`, async() => {
+      await yamlView.saveButton.click();
+      expect(crudView.errorMessage.isPresent()).toBe(false);
+    });
+
+    it(`displays detail view for ${crd} instance`, async() => {
+      await browser.wait(until.presenceOf(crudView.resourceTitle));
+      expect(browser.getCurrentUrl()).toContain(`/${name}`);
+      expect(crudView.resourceTitle.getText()).toEqual(name);
+    });
+
+    it(`creates a new test pod to display the ${crd} instance`, async() => {
+      await browser.get(`${appHost}/k8s/ns/${testName}/pods`);
+      await crudView.isLoaded();
+      await crudView.createYAMLButton.click();
+      await yamlView.isLoaded();
+      const content = await yamlView.getEditorContent();
+      const newContent = _.defaultsDeep({}, {metadata: {name: podName}}, safeLoad(content));
```

Comment:
Does this fix the tests? It looks like there's another test not deleting a workload that's adopting and killing your pod.

```suggestion
      const newContent = _.defaultsDeep({}, {metadata: {name: podName, labels: {app: example}}}, safeLoad(content));
```